### PR TITLE
Support Incidents API

### PR DESCRIPTION
**Pull request by GeneticGenesis**
* Support Incidents API.
* Bump minor version.

### CODE REVIEW

File: MuxPhp/Api/IncidentsApi.php [hunk 1]
```diff
@@ -0,0 +1,1040 @@
...
+
+    /**
+     * Operation listIncidents
+     *
+     * List Incidents
+     *
+     * @param  mixed[] $optionalParams An associative array of optional parameters which can be passed to this function:
+     *     - limit int - Number of items to include in the response (optional, default to 25)
+     *     - page int - Offset by this many pages, of the size of &#x60;limit&#x60; (optional, default to 1)
+     *     - order_by string - Value to order the results by (optional)
```
Comment (jsanford8) [inline MuxPhp/Api/IncidentsApi.php]: Do we not have the valid options listed in other SDKs/other routes? If not, no need to add here, but might be nice to have for valid options.

File: MuxPhp/Models/Incident.php [hunk 2]
```diff
@@ -0,0 +1,873 @@
...
+     *
+     * @var string[]
+     */
+    protected static $openAPITypes = [
+        'threshold' => 'double',
+        'status' => 'string',
+        'started_at' => 'string',
+        'severity' => 'string',
+        'sample_size_unit' => 'string',
+        'sample_size' => 'double',
```
Comment (jsanford8) [inline MuxPhp/Models/Incident.php]: Really? I thought this would be an integer

File: MuxPhp/Models/Incident.php [hunk 3]
```diff
@@ -0,0 +1,873 @@
...
+        'measured_value' => 'double',
+        'incident_key' => 'string',
+        'impact' => 'string',
+        'id' => 'string',
+        'error_description' => 'string',
+        'description' => 'string',
+        'breakdowns' => '\MuxPhp\Models\IncidentBreakdown[]',
+        'affected_views_per_hour_on_open' => 'double',
+        'affected_views_per_hour' => 'double',
+        'affected_views' => 'double'
```
Comment (jsanford8) [inline MuxPhp/Models/Incident.php]: This should also be an int, right?

File: MuxPhp/Models/Incident.php [hunk 4]
```diff
@@ -0,0 +1,873 @@
...
+        'measured_value' => 'double',
+        'incident_key' => null,
+        'impact' => null,
+        'id' => null,
+        'error_description' => null,
+        'description' => null,
+        'breakdowns' => null,
+        'affected_views_per_hour_on_open' => 'int64',
+        'affected_views_per_hour' => 'int64',
+        'affected_views' => 'int64'
```
Comment (jsanford8) [inline MuxPhp/Models/Incident.php]: Oh so it is, kind of, but PHP needs it to be a double above?

File: MuxPhp/Models/Incident.php [hunk 5]
```diff
@@ -0,0 +1,873 @@
...
+        return self::$openAPIFormats;
+    }
+
+    /**
+     * Array of attributes where the key is the local name,
+     * and the value is the original name
+     *
+     * @var string[]
+     */
+    protected static $attributeMap = [
```
Comment (jsanford8) [inline MuxPhp/Models/Incident.php]: is this... something that was generated? Seems kind of useless

File: MuxPhp/Models/IncidentNotification.php [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+    protected static $openAPIModelName = 'IncidentNotification';
+
+    /**
+     * Array of property to type mappings. Used for (de)serialization
+     *
+     * @var string[]
+     */
+    protected static $openAPITypes = [
+        'queued_at' => 'string',
+        'id' => 'double',
```
Comment (jsanford8) [inline MuxPhp/Models/IncidentNotification.php]: :( we made this a number didn't we... still not sure why `double` here, but seems like just how it's done.

**Comment by GeneticGenesis** [comment]
We've backed out incidents support for now, we'll pick this up again soon.

**Pull request closed without merging**